onnxruntime/test/onnx/TestCase.cc

```diff
@@ -27,6 +27,8 @@ using namespace onnxruntime;
 using namespace onnxruntime::common;
 using google::protobuf::RepeatedPtrField;
 
+static constexpr int protobuf_block_size = 1 << 20;
```

Comment:
>protobuf_block_size [](start = 21, length = 19)

In KByes?

Reply:
Bytes. 1<<20 is 1MB.

Reply:
See #4020

Reply:
how about `protobuf_block_size_in_bytes`?